pom.xml

```diff
@@ -61,7 +61,7 @@
         <exec-maven-plugin.version>1.5.0</exec-maven-plugin.version>
         <spotbugs.version>4.7.1</spotbugs.version>
         <spotbugs.maven.plugin.version>4.7.1.0</spotbugs.maven.plugin.version>
-        <java.version>8</java.version>
+        <java.version>17</java.version>
```

Comment:
I thought it is supposed to be java 11?

Reply:
Based on https://github.com/confluentinc/common/pull/707 I think it's 17